src/widget.ts

```diff
@@ -69,12 +71,24 @@ export class MapModel extends DOMWidgetModel {
 export class MapView extends DOMWidgetView {
   render() {
     useGeographic();
-    this.el.classList.add('custom-widget');
-
-    this.mapContainer = document.createElement('div');
-    this.mapContainer.style.height = '500px';
-    this.mapContainer.style.width = '100%';
-    this.el.appendChild(this.mapContainer);
+    this.el.classList.add('jupyter-widgets');
+    this.el.classList.add('ipyopenlayer-widgets');
+
+    this.map_container = document.createElement('div');
+    this.map_container.classList.add('ol-container');
+    requestAnimationFrame(() => {
```

Comment:
It's unclear to me why we need a `requestAnimationFrame` here, what was the reason for adding it?

Reply:
I was trying to get the parentelement with this : this.el.parentElement but it was returning null at first, So I added requestAnimationFrame to give it more time to get executes and to get the right answer.